tests/FSharp.Build.UnitTests/FSharp.Build.UnitTests.fsproj

```diff
@@ -11,6 +11,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\FSharp.Test.Utilities\XunitSetup.fs">
+      <Link>XunitSetup.fs</Link>
+    </Compile>
```

Comment:
This file needs to be included in a each project that uses `TestConsole` as an assembly fixture.